linux/debian/8/foxpass_setup.py

```diff
@@ -49,9 +51,9 @@ def main():
 
     apt_get_update()
     install_dependencies()
-    write_foxpass_ssh_keys_script(args.api_url, args.api_key)
```

Comment:
combine args.api_url and args.apis into one array to send into write_foxpass_ssh_keys_script()